node/rpc/src/lib.rs

```diff
@@ -38,3 +44,27 @@ mod constants {
 	/// This typically means that the runtime trapped.
 	pub const RUNTIME_ERROR: i64 = 1;
 }
+
+/// Instantiate all RPC extensions.
+pub fn create<C, P, M>(client: Arc<C>, pool: Arc<Pool<P>>) -> jsonrpc_core::IoHandler<M> where
```

Comment:
If we expose only `create`, should we then unmark `self::{accounts, countracts}` as `pub`?

Reply:
I suppose `account` RPCs are still valuable on runtimes with no contracts support and vice-versa, right? That will allow people to import `node-rpc` and just instantiate one.

Reply:
Ah, yeah, good point. Although I didn't realize that `node` is a part of public API.